internal/collectors/cpu.go

```diff
@@ -115,3 +124,51 @@ func addArm64Extras(result Result) Result {
 	}
 	return result
 }
+
+// NOTE: Z systems support
+
+func exactIntMatch(layerID string, text []byte) (int, error) {
```

Comment:
Not really, because we are trying to match exactly an integer value, but this integer value is not an index on anything. In this case, it's the number of CPUs as returned by the `read_values` tool.